### DISCUSSION TITLE
SuperPoint Open Support

### DISCUSSION THREAD

**Discussion by flashcolab** [fabio-sim/LightGlue-ONNX#64] [Ideas]
Can we expect a future release to have Support for SuperPoint Open?

**Comment by fabio-sim** [comment]
Hi @flashcolab, thank you for your interest in LightGlue-ONNX.

Duplicate of #39.
Also see https://github.com/cvg/glue-factory/issues/22 and https://github.com/cvg/LightGlue/issues/96
